src/styles/parts/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  width: 200px;
+  height: 400px;
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  border: 1 solid #f3f3f3;
+  border-radius: 5;
+
+  &__image {
+    height: 134;
+    width: 160;
+    margin-bottom: 40;
```

Comment:
The margin-bottom value is missing a unit. It should be specified, for example, as '40px'.